web/packages/teleport/src/UnifiedResources/FilterPanel.tsx

```diff
@@ -23,41 +23,39 @@ import Menu, { MenuItem } from 'design/Menu';
 import { StyledCheckbox } from 'design/Checkbox';
 import { ArrowUp, ArrowDown, ChevronDown } from 'design/Icon';
 
-import { encodeUrlQueryParams } from 'teleport/components/hooks/useUrlFiltering';
 import { ResourceFilter, SortType } from 'teleport/services/agents';
 
 import { HoverTooltip } from './Resources';
-
-const kindOptions = [
-  { label: 'Application', value: 'app' },
-  { label: 'Database', value: 'db' },
-  { label: 'Desktop', value: 'windows_desktop' },
-  { label: 'Kubernetes', value: 'kube_cluster' },
-  { label: 'Server', value: 'node' },
-];
+import { SharedUnifiedResource } from './types';
+
+const kindToLabel: Record<SharedUnifiedResource['resource']['kind'], string> = {
+  app: 'Application',
+  db: 'Database',
+  windows_desktop: 'Desktop',
+  kube_cluster: 'Kubernetes',
+  node: 'Server',
+  user_group: 'User group',
+  windows_desktop_service: 'Windows desktop service',
```

Comment:
`windows_desktop_service` isn't used as a kind in the unified cache, we can remove it.

Reply:
In that case, I can remove it completely from the `SharedUnifiedResource` type. 
The only problem is that I won't be able to assign [teleport's `UnifiedResource`](https://github.com/gravitational/teleport/blob/618bb5a81ed90f585d6ab6c1976641e3dd530588/web/packages/teleport/src/services/agents/types.ts#L33) to the shared type, so probably I should remove it from there as well. 
So if I'm correct, that type was wrong from the beginning?

Reply:
Yeah seems like it was wrong from the beginning!

Reply:
Thanks, removed!
I also had to fix it in `e`, it uses that incorrect type https://github.com/gravitational/teleport.e/pull/2492.